app/views/groups/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<section>
+  <header class="bg-blue-500">
+    <nav class="h-16 flex justify-between items-center max-w-sm md:max-w-lg lg:max-w-2xl xl:max-w-4xl mx-auto">
+      <%= image_tag('menu.svg', alt: 'Menu', class: "w-5 h-5") %>
+      <h2 class="text-2xl text-white font-semibold">Categories</h2>
+      <%= link_to "Log out", destroy_user_session_path, method: :delete, data: { turbo_method: :delete }, class: "text-md text-white" %>
+    </nav>
+  </header>
+  <article class="flex flex-col gap-y-4 max-w-sm md:max-w-lg lg:max-w-2xl xl:max-w-4xl my-4 mx-auto">
+    <% @groups.each do |group| %>
+      <%= link_to group_path(group.id), class: 'details_link' do %>
+      <div class="flex gap-x-4 border border-gray-200 p-2 bg-white shadow-lg">
+        <img class="w-20 h-20" src=<%= group.icon %> >
+        <div class="flex justify-between w-full my-2">
+          <div class="flex flex-col justify-between">
+            <h2 class="text-lg font-bold"><%= group.name %></h2>
+            <p><%= group.created_at.strftime('%m-%d-%Y') %></p>
+          </div>
+          <p class="font-bold">$<%= group.operations.sum { |op| op.amount } %></p>
+        </div>
+      </div>
+      <% end %>
+    <% end %>
+    <div class="flex justify-center">
+      <%= link_to new_group_path, class: 'bg-green-500 px-4 py-1 rounded text-white font-semibold' do %>
+        Add new Category
+      <% end %>
+    </div>
```

Comment:
- [ ] The new category button should be placed at the bottom page and the width should spread across the page as contained in the design.

  ![image](https://github.com/Felipe-Perez-Ferraro/trans_by/assets/81468675/9bfd2979-d955-438e-8e44-a7e4251b8389)

  ![image](https://github.com/Felipe-Perez-Ferraro/trans_by/assets/81468675/70b24889-d3ee-4c00-8ed3-a351a387a36f)